src/panels/lovelace/editor/types.ts

```diff
@@ -10,3 +11,10 @@ export interface ConfigValue {
   format: "js" | "yaml";
   value: string | LovelaceConfig;
 }
+
+export interface EditorEvent {
```

Comment:
This interface seems very entity specific, should it maybe be called `EntitiesEditorEvent` or something like that?